rust/tests/integration/producer.rs

```diff
@@ -636,6 +630,7 @@ fn produce_already_used_mid_ssrc() {
                     parameters.encodings = vec![RtpEncodingParameters {
                         ssrc: Some(22222222),
                         rtx: Some(RtpEncodingParametersRtx { ssrc: 6667 }),
+                        dtx: Some(false),
```

Comment:
Why did this need to change from `None` to `Some(false)`?

Reply:
Will put it back to None. Option<boolean> already seems a bit rare to begin with.